src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
+        String username = "";
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login("bob", "12345");
+            username = user.getLogin();
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't login a user: " + username, e);
```

Comment:
The variable `username` is initialized to an empty string and is used in the error log message. If the login fails, `username` will remain an empty string, which might not provide useful information in the log. Consider logging the username that was attempted to be used for login, if available, or adjust the log message to reflect the lack of a username.